tests/integration-tests/tests/proxy/test_proxy.py

```diff
@@ -74,7 +74,7 @@ def get_instance_public_ip(instance_id, region):
 
 
 @pytest.mark.usefixtures("region", "os", "instance", "scheduler")
-def test_proxy(pcluster_config_reader, clusters_factory, proxy_stack_factory, scheduler_commands_factory):
```

Comment:
Even better, clusters_factory could be between proxy_stack_factory and proxy_stack_factory

Reply:
You mean proxy_stack_factory and scheduler_commands_factory, right?
Done!